cql/fhir-cql/src/main/java/com/ibm/fhir/cql/engine/retrieve/SearchParameterFHIRRetrieveProvider.java

```diff
@@ -134,30 +134,30 @@ public int getMaxCodesPerQuery() {
         return result;
     }
 
-    protected Pair<String, IQueryParameter> getTemplateParam(String dataType, String templateId) {
+    protected Pair<String, IQueryParameter> getTemplateParam(String resourceType, String templateId) {
         // purposefully empty
         return null;
     }
 
-    protected Pair<String, IQueryParameter> getContextParam(String dataType, String context, String contextPath,
-        Object contextValue) throws Exception {
+    protected Pair<String, IQueryParameter> getContextParam(String resourceType, String context, String contextPath,
+            Object contextValue) throws Exception {
         Pair<String, IQueryParameter> result = null;
 
         if (context != null && "Patient".equals(context) && contextValue != null && contextPath != null) {
-            result = searchParameterResolver.createSearchParameter(context, dataType, contextPath, (String) contextValue);
+            result = searchParameterResolver.createSearchParameter(context, resourceType, contextPath, (String) contextValue);
             if (result == null) {
-                throw new IllegalArgumentException(String.format("Could not resolve search parameter for dataType '%s' and contextPath '%s'", dataType, contextPath));
+                throw new IllegalArgumentException(String.format("Could not resolve search parameter for dataType '%s' and contextPath '%s'", resourceType, contextPath));
```

Comment:
Change message from 'dataType' to 'resourceType'.

Reply:
Yeah, I was on the fence about whether committing this 'dataType' -> 'resourceType' change.
I made that change locally because 'dataType' was throwing me off (in FHIR you can only search on resourceType).

However, the 'dataType' name is what gets used in the interface here:  https://github.com/DBCG/cql_engine/blob/master/engine/src/main/java/org/opencds/cqf/cql/engine/retrieve/RetrieveProvider.java#L7

And, obviously, thats not under our control.
I guess I'll update the messages to say 'resourceType' for now, since thats definitely what we'll be working with in this search-based impl.